### PR TITLE
Use Composer scripts to introduce composer test

### PR DESCRIPTION
**Pull request by RobLoach**
https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

This allows running tests via `composer test`.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -73,5 +73,8 @@
     "require-dev": {
         "composer/composer": "1.0.*@dev",
         "phpunit/phpunit": "4.1.*"
+    },
+    "scripts": {
+        "test": "phpunit"
```
Comment (potherca) [inline composer.json]: Since PhpUnit is a dev requirement, shouldn't this point to `vendor/bin/phpunit` rather than just `phpunit`?

**Comment by RobLoach** [comment]
> Since PhpUnit is a dev requirement, shouldn't this point to vendor/bin/phpunit rather than just phpunit?

Composer adds vendor/bin to the PATH, so you don't have to add it in yourself :wink: .

**Comment by potherca** [comment]
Cool! It didn't use to, do you have a reference when this was added? (That's the down-side of being an early adapter, if you don't keep reading up on changes, you quickly fall under the [Law of the handicap of a head start](http://en.m.wikipedia.org/wiki/Law_of_the_handicap_of_a_head_start))

**Comment by RobLoach** [comment]
https://getcomposer.org/doc/articles/scripts.md#writing-custom-commands

> Note: Composer's bin-dir is pushed on top of the PATH so that binaries of dependencies are easily accessible as CLI commands when writing scripts.

Not sure when it was added, however.

> [Law of the handicap of a head start](http://en.m.wikipedia.org/wiki/Law_of_the_handicap_of_a_head_start)

Hah, didn't know that one. Good find.

**Comment by potherca** [comment]
Judging by the docs commit log it was added Jul 23, 2014. Thanks for the reference ;-)

**Pull request merged**